### ISSUE TITLE
How can i use custom dataset to fine-tuning caption task?

### ISSUE THREAD

**Issue by zml110120** [OFA-Sys/OFA#150]
Dear author，
It is an awesome job！
I want to use a custom data set for caption Task. I want to know how to generate BPE related  files and 【coco-valid-words.p】 files using custom data sets. I have generated TSV files for the training.

![image](https://user-images.githubusercontent.com/29172684/176823011-bb2394a4-56b8-4f6b-a2c0-2c732261a544.png)

**Comment by logicwong** [comment]
@zml110120 Are you using English data? If so, you just need to use the default BPE file we provided. 

`coco-valid-words.p` contains two items: 
1. ref_len: the number of images in the validation set; 
2. document_frequency: the document frequency of each word. 

You can refer to `utils/cider/pyciderevalcap/ciderD/ciderD_scorer.py` to construct this file
![41711974-A7FD-44E0-ACA3-A619572F741B](https://user-images.githubusercontent.com/36780733/176832620-1cc20435-cfdf-4c56-ab24-6a5e0863fff0.png)

**Comment by zml110120** [comment]
That's right. I'm using data in English. Now I only have a Json file in COCO format. I don't have any other files, like 【Pickle】. How do I generate 【coco-valid-words.p】?

**Comment by zml110120** [comment]
@logicwong That's right. I'm using data in English. Now I only have a Json file in COCO format. I don't have any other files, like 【Pickle】. How do I generate 【coco-valid-words.p】?

**Comment by logicwong** [comment]
@zml110120 `coco-valid-words.p` is used to compute CIDEr score on the validation set during validation. If you don't need this feature, you can remove `--eval-cider`, `--eval-cider-cached-tokens`, and `--eval-args`.
